integrations.go

```diff
@@ -0,0 +1,105 @@
+package sentry
+
+import (
+	"bufio"
+	"io/ioutil"
+  "os"
+  "fmt"
+	"encoding/json"
+	"strings"
+)
+
+func GetModules() (map[string]string, error) {
```

Comment:
Can we split this into 3 private functions, one for each possible scenario? It'd make `GetModules` much easier to read.

```go
func GetModules() (map[string]string, error) {
  if something {
    return _getModulesOneWay()
  }

  if somethingElse {
    return _getModulesOtherWay()
  }

  return _defaultWay()
}
```

Reply:
For sure.